public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -32,6 +45,7 @@ export interface Props {
   onClear?: () => void;
   accessControlMetadata?: boolean;
   customAdd?: boolean;
+  dissalowSlashes?: boolean;
```

Comment:
What if we used something like `customErrorMessage` that would enable setting different error messages (or other informative things). Then we could move the slashes-specific code into the alerting codebase and use the `customErrorMessage` prop as an injection point to FolderPicker

Reply:
I may be also `customValidator` or something similar

Reply:
100% agree with this idea, I'm going to think about it